File: 6. Sum Square Difference/solution.ts
function sumOfSquares(n: number): number {
    return Math.floor((n * (n + 1) * (2 * n + 1)) / 6);
}

function squareOfSum(n: number): number {
    return Math.floor((n * (n + 1)) / 2) ** 2;
}

const n = 100
const difference = squareOfSum(n) - sumOfSquares(n)

console.log(difference);

File: 6. Sum Square Difference/solution.py
def sum_of_squares(n: int) -> int:
    return (n * (n + 1) * (2 * n + 1)) // 6

def square_of_sum(n: int) -> int:
    return ((n * (n + 1)) // 2) ** 2

n = 100
difference = square_of_sum(n) - sum_of_squares(n)

print(difference)